code/lib/preview-api/src/modules/preview-web/docs-context/DocsRenderFunction.ts

```diff
@@ -0,0 +1,3 @@
+import type { DocsRenderFunction } from '@storybook/types';
+
+export type { DocsRenderFunction };
```

Comment:
For curiosity. Is a re-export for types possible?

```suggestion
export type { DocsRenderFunction } from '@storybook/types';
```

Reply:
Yeah, I'm pretty sure that's possible.